.github/workflows/cli-build-instructions.yml

```diff
@@ -36,17 +36,17 @@ jobs:
             ${{ runner.os }}-maven-
       - name: Set up in-source output dir
         run: |
-          mkdir -p output/Generated/BIN
+          mkdir -p Generated/BIN
```

Comment:
side note: I accidentally clicked 'rich diff' on THIS file.  The results were unexpected but potentially helpful.